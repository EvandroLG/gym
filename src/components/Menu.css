ul.nav {
  display: flex;
}

ul.nav li {
  margin-right: 1%;
}

ul.nav li:last-child {
  margin-right: 0;
}

ul.nav a {
  color: #03a9f4;
}
