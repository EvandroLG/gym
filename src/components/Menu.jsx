import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';

import './Menu.css';

export default class Menu extends Component {
  render() {
    return (
      <ul className="nav">
        <li>
          <NavLink exact to="/" activeClassName="disabled">Training</NavLink>
        </li>
        <li>
          <NavLink to="/new_training" activeClassName="disabled">New Training</NavLink>
        </li>
      </ul>
    )
  }
}
