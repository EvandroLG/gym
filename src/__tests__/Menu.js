import React from 'react';
import { shallow } from 'enzyme';
import Menu from '../components/Menu';

/* global verifySnapshot */

describe('menu', () => {

  it('component should render as expected', () => {
    const component = shallow(<Menu />);
    verifySnapshot(component);
  });
});
