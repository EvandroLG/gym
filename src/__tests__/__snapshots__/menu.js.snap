// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`menu component should render as expected 1`] = `
<ul
  className="nav"
>
  <li
    className="nav-item"
  >
    <NavLink
      activeClassName="disabled"
      ariaCurrent="true"
      className="nav-link"
      exact={true}
      to="/"
    >
      Training
    </NavLink>
  </li>
  <li
    className="nav-item"
  >
    <NavLink
      activeClassName="disabled"
      ariaCurrent="true"
      className="nav-link"
      to="/new_training"
    >
      New Training
    </NavLink>
  </li>
</ul>
`;
